plugins/janus_streaming.c

```diff
@@ -79,11 +79,14 @@ videobufferkf = yes|no (whether the plugin should store the latest
 	keyframe and send it immediately for new viewers, EXPERIMENTAL)
 databuffermsg = yes|no (whether the plugin should store the latest
 	message and send it immediately for new viewers)
+x-audio-multicast-iface = network interface IP address or device name to listen on when receiving multicast audio
+x-video-multicast-iface = network interface IP address or device name to listen on when receiving multicast video
```

Comment:
Please use a naming that is consistent with the others, e.g., `audioiface`. Besides, any reason why this should be limited to multicast, and not apply to unicast as well? There may be several reasons to limit the interfaces a mountpoint listens on (e.g., to make sure only another process in your lan can feed a mountpoint, and not anybody on the internet who knows the port). As a side note, there should be one for data channels too, as they're also a possible property in mountpoints now.